app/src/main/java/telenav/demo/app/utils/CategoryAndFiltersUtil.kt

```diff
@@ -72,6 +185,14 @@ object CategoryAndFiltersUtil {
 
         when (ratingLevel) {
 
+            -1.0 -> {
+                firstStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
+                secondStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
+                thirdStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
+                fourthStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
+                fifthStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
+            }
+
```

Comment:
It hides stars in case of rating value is 0.0 

            0.0 -> {
                firstStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
                secondStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
                thirdStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
                fourthStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
                fifthStar.setImageDrawable(context.getDrawable(R.drawable.ic_star_empty))
            }